components/BudgetItemsList.js

```diff
@@ -369,8 +376,13 @@ const BudgetItem = ({ item, isInverted, isCompact, canDownloadInvoice, intl }) =
               )}
             </Container>
           </Flex>
-          <Container fontSize="LeadParagraph" ml="auto" order={[2, 3]}>
-            <Span color={isCredit ? 'green.700' : 'red.700'} mr={2} css={{ verticalAlign: 'text-bottom' }}>
+          <Container data-cy="transaction-amount" fontSize="LeadParagraph" ml="auto" order={[2, 3]}>
+            <Span
+              data-cy="credit"
```

Comment:
Name is not great, this DOM node is about the transaction type sign - which can either be positive or negative (debit or credit). What about the following one:

```suggestion
              data-cy="transaction-sign"
```